.eslintrc

```diff
@@ -8,7 +8,8 @@
     "plugin:jest/recommended"
   ],
   "globals": {
-    "__": true
+    "__": true,
+    "Jed": true
```

Comment:
Is Jed not in webpack?  Can we not import it instead?

Reply:
We currently share Jed with other js code outside of our webpack stack and populate the translations into `window.Jed`. We could import it just to use its sprintf function.